stable/redis-ha/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: redis-ha
+home: http://redis.io/
+engine: gotpl
+keywords:
+- redis
+- keyvalue
+- database
+version: 0.2.0
+description: Highly available Redis cluster with multiple sentinels and standbys.
+maintainers:
+  - email: ikaboubi@gmail.com
+    name: KABOUBI
+details:
+  This package provides a highly available Redis cluster with multiple sentinels and standbys.
+  Note the `redis-master` pod is used for bootstrapping only and can be deleted once
+  the cluster is up and running.
```

Comment:
Is this true? If I delete the `redis-master` StatefulSet I'll be removing one of two redis nodes and one of three sentinels in which case sentinel can't build consensus to promote the new master and there's only one replica left running.